fastlane_core/lib/fastlane_core/cert_checker.rb

```diff
@@ -63,8 +63,22 @@ def self.install_wwdr_certificate
       filename = file.path
       keychain = wwdr_keychain
       keychain = "-k #{keychain.shellescape}" unless keychain.empty?
-      Helper.backticks("curl -f -o #{filename} #{url} && security import #{filename} #{keychain}", print: FastlaneCore::Globals.verbose?)
-      UI.user_error!("Could not install WWDR certificate") unless $?.success?
+
+      require 'open3'
+      stdout, stderr, status = Open3.capture3("curl -f -o #{filename} #{url} && security import #{filename} #{keychain}")
+      if FastlaneCore::Globals.verbose?
+        UI.command_output(stdout)
+        UI.command_output(stderr)
+      end
+
+      unless status.success?
+        UI.verbose("Failed to install WWDR Certificate, checking output to see why")
+        # Check the command output, WWDR might already exist
+        unless /The specified item already exists in the keychain./ =~ stderr
```

Comment:
Missing a link to the issue with some context

Reply:
Link to issue is in PR description and commit 🤔